twill11/hometask-3/lib/validator.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Validate errors
+class Validator
+  def initialize(town)
+    @town = town
+    @errors = []
+  end
+
+  def validate
+    add_error('City name can not be blank!') if empty?
+  end
+
+  def valid?
+    error_messages.empty?
+  end
```

Comment:
You should not use error_message in this place. This class has already another variable that is more suitable for it.